src/EditorFeatures/Test/TestExportProvider.cs

```diff
@@ -110,11 +104,10 @@ private static Type[] GetNeutralAndCSharpAndVisualBasicTypes()
                 typeof(CodeAnalysis.VisualBasic.CodeGeneration.VisualBasicSyntaxGenerator),
                 typeof(CSharp.LanguageServices.CSharpContentTypeLanguageService),
                 typeof(VisualBasic.LanguageServices.VisualBasicContentTypeLanguageService),
-                typeof(IncrementalCaches.SymbolTreeInfoIncrementalAnalyzerProvider),
-                typeof(CodeAnalysis.Diagnostics.EngineV2.InProcCodeAnalysisDiagnosticAnalyzerExecutor)
+                typeof(TestExportProvider)
             };
 
-            return MinimalTestExportProvider.GetLanguageNeutralTypes()
+            return ServiceTestExportProvider.GetLanguageNeutralTypes()
```

Comment:
keeping TestExportProvider here since all kind of MEF exports are dumped here, it is impossible to know which tests actually depend on what MEF exports and where those are defined.